accelerator/config.c

```diff
@@ -54,7 +54,7 @@ status_t cli_core_set(ta_core_t* const core, int key, char* const value) {
       ta_conf->host = value;
       break;
     case TA_PORT_CLI:
-      ta_conf->port = value;
+      ta_conf->port = atoi(value);
```

Comment:
The behavior of `atoi` is the same as
```c
strtol(nptr, NULL, 10);
```
except that atoi() does not **detect errors**. Thus, there is potential risk if the user assigns wrong input string.